src/ngamsUtils/setup.py

```diff
@@ -0,0 +1,60 @@
+#
+#    ICRAR - International Centre for Radio Astronomy Research
+#    (c) UWA - The University of Western Australia, 2015
+#    Copyright by UWA (in the framework of the ICRAR)
+#    All rights reserved
+#
+#    This library is free software; you can redistribute it and/or
+#    modify it under the terms of the GNU Lesser General Public
+#    License as published by the Free Software Foundation; either
+#    version 2.1 of the License, or (at your option) any later version.
+#
+#    This library is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#    Lesser General Public License for more details.
+#
+#    You should have received a copy of the GNU Lesser General Public
+#    License along with this library; if not, write to the Free Software
+#    Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+#    MA 02111-1307  USA
+#
+import os
+import sys
+
+from setuptools import setup, find_packages
+
+with open('../../VERSION') as vfile:
+    for line in vfile.readlines():
+        if "ngamsNGAMS_SW_VER" in line:
+            version = line.split("NGAMS_SW_VER ")[1].strip()[1:-1]
+            break
+
+setup(
+    name='ngamsUtils',
+    version=version,
+    description="The python package containing the NGAS utility python scripts",
+    long_description="The python package containing the NGAS utility python scripts",
+    classifiers=[], # Get strings from http://pypi.python.org/pypi?%3Aaction=list_classifiers
+    keywords='',
+    author='',
+    author_email='',
+    url='',
+    license='LGPL',
+    packages=find_packages(),
+    include_package_data=True,
+    package_data={
+        'ngamsUtils' : ['README']
+    },
+    install_requires=['ngamsCore'],
+    entry_points= {
+        'console_scripts':[
+            'ngas-check-file-list=ngamsUtils.ngasCheckFileList:main',
+            'ngas-check-file-copies=ngamsUtils.ngasCheckFileCopies:main',
+            'ngas-discard-files=ngamsUtils.ngasDiscardFiles:main',
+            'ngas-verify-cloning=ngamsUtils.ngasVerifyCloning:main',
+            'ngas-xsync-tool=ngamsUtils.ngasXSyncTool:main'
```

Comment:
What I mentioned privately in our email exchange is that here I would have declared the script names unchanged (to avoid breaking existing third-party scripts that rely on the old script names), but renamed the modules. Something like `ngasCheckFileList=ngamsUtils.check_file_list:main`. If you think renaming the scripts is not an issue then that's fine, but please double-check that's the case.

Reply:
I would assume after several years of neglect no one is using the utility scripts. There were completely broken.
I have kept the original module names. It is only the executable name that I changed. I try and following Unix convention and avoid using camel case for naming executables. I don't see why this should break anything. The AOG use the scripts from the command line. I am confident they can adapt to the new names.

Reply:
If you're confident the new names won't break anyone's code then sure, I also vote for the new ones (I also like them better). And in any case we can always add back the old names in case somebody complains really loudly.